dist/spec.yaml

```diff
@@ -10693,7 +10694,7 @@ definitions:
       x5t:
         readOnly: false
         type: string
-      'x5t#S256':
+      x5t#S256:
```

Comment:
Is this change correct?

Reply:
I think this change is generated automatically depending on your env.

Reply:
Yeah, happens after running yarn build. I assumed it was by design.